.github/workflows/release.yml

```diff
@@ -16,34 +17,9 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  create-release:
-    name: create github & crates.io release
-    #if: github.repository-owner == 'zeskeertwee'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          persist-credentials: false
-      - uses: taiki-e/github-actions/install-rust@main
-        with:
-          toolchain: nightly
-      - run: cargo test --verbose -- --test-threads=1
-      - run: cargo package -p vach
-      - run: cargo package -p vach-cli
-      - uses: taiki-e/create-gh-release-action@v1
-        with:
-          title: $version
-          branch: main
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This part is needed for the github release since it creates the release for the binaries to be appended to

Reply:
I fixed this long ago, now we have to make sure the workflow only runs on [`zeskeertwee:main`](https://github.com/zeskeertwee/vach)